Doc/library/typing.rst

```diff
@@ -2651,6 +2652,35 @@ Functions and decorators
    runtime we intentionally don't check anything (we want this
    to be as fast as possible).
 
+.. decorator:: copy_kwargs(source_func)
+
+    Cast the decorated function's call signature to the *source_func*'s.
```

Comment:
reStructuredText uses three spaces for indents in most places, see https://devguide.python.org/documentation/markup/#use-of-whitespace. Please re-indent this section.